api/config_entry.go

```diff
@@ -218,21 +218,22 @@ type UpstreamLimits struct {
 }
 
 type ServiceConfigEntry struct {
-	Kind             string
-	Name             string
-	Partition        string                  `json:",omitempty"`
-	Namespace        string                  `json:",omitempty"`
-	Protocol         string                  `json:",omitempty"`
-	Mode             ProxyMode               `json:",omitempty"`
-	TransparentProxy *TransparentProxyConfig `json:",omitempty" alias:"transparent_proxy"`
-	MeshGateway      MeshGatewayConfig       `json:",omitempty" alias:"mesh_gateway"`
-	Expose           ExposeConfig            `json:",omitempty"`
-	ExternalSNI      string                  `json:",omitempty" alias:"external_sni"`
-	UpstreamConfig   *UpstreamConfiguration  `json:",omitempty" alias:"upstream_config"`
-	Destination      *DestinationConfig      `json:",omitempty"`
-	Meta             map[string]string       `json:",omitempty"`
-	CreateIndex      uint64
-	ModifyIndex      uint64
+	Kind                  string
+	Name                  string
+	Partition             string                  `json:",omitempty"`
+	Namespace             string                  `json:",omitempty"`
+	Protocol              string                  `json:",omitempty"`
+	Mode                  ProxyMode               `json:",omitempty"`
+	TransparentProxy      *TransparentProxyConfig `json:",omitempty" alias:"transparent_proxy"`
+	MeshGateway           MeshGatewayConfig       `json:",omitempty" alias:"mesh_gateway"`
+	Expose                ExposeConfig            `json:",omitempty"`
+	ExternalSNI           string                  `json:",omitempty" alias:"external_sni"`
+	UpstreamConfig        *UpstreamConfiguration  `json:",omitempty" alias:"upstream_config"`
+	Destination           *DestinationConfig      `json:",omitempty"`
+	MaxInboundConnections int
```

Comment:
Should this be an `*int` w/ `omitempty` to be backward compatible? Like this one: https://github.com/hashicorp/consul/blob/0e2eed12a2ef9e8c2b9b671a5112730f15531d59/agent/structs/config_entry.go#L991-L992

Reply:
Although I see the `structs` package doesn't have this 🤔